### DISCUSSION TITLE
CDN implementation doesn't work

### DISCUSSION THREAD

**Discussion by jcrist1027** [luwes/playerx#56] [Q&A]
Beat my head against the wall for several minutes then hit the CodeSandbox link and the CDN implementation does not appear to work.

https://22ku4.csb.app/

**Comment by luwes** [comment]
Hi, seems to work on my end. https://codesandbox.io/s/hello-playerx-22ku4
Is this not showing a Vimeo player? Any errors in the console?

**Comment by jcrist1027** [comment]
Initially I saw some js was blocked by my ad blocker when first checking
just now. I turned it off and did an F5 refresh and the blocking is gone.
A few typical warnings. But I just get a blank page.

[image: image.png]

**Comment by jcrist1027** [comment]
Same behavior if I go to the sandbox URL on my Samsung Phone Chrome
Browser.

I initially try implementing the lite-vimeo CDN code.  That didn't work
either. So it looks to be a lite-vimeo via CDN script load issue I'm
guessing.  Do that wouldn't be your area of responsibility unless you can
tell me why lite-vimeo isn't working to pass that along to slightlyoff.

Checked on Firefox too which I run no plugins on.  Maybe we aren't looking
at the exact same code or your browser had the code cached when it worked?
Not sure.  Strange that you would see the video and the sandbox preview
doesn't work on my desktop or my mobile phone. I'm not behind any sort of
network firewall that would block it.

[image: image.png]

**Reply by cwsqer006** [reply]
![Oprah Crazy](https://media.giphy.com/media/kPTl08vSEcoAnsTvy8/giphy.gif)lauug

**Comment by jcrist1027** [comment]
Your demo page works.  But you're not loading using the CDN <script> method
I would assume.

[image: image.png]

**Reply by luwes** [reply]
it's using the unpkg CDN `<script src="https://unpkg.com/playerx"></script>`

**Comment by cwsqer006** [comment]
happy![Leo Clap](https://media.giphy.com/media/gLu90OMjz4j3hAkJk2/giphy.gif)
